app/controllers/user/dashboard_controller.rb

```diff
@@ -1,3 +1,7 @@
 class User::DashboardController < ApplicationController
-  def show; end
+  def show
+    cu_id = current_user.id
+    @friends = User.where(id: Friendship.where(user_id: cu_id).pluck(:friend_id)).or(User.where(id: Friendship.where(friend_id: cu_id).pluck(:user_id)))
```

Comment:
I was able to get this one down to one line!  But this is the line Rubocop thinks is too long (which, honestly, I get it).